web/src/components/ProfessionalSummary/ProfessionalSummary.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import styles from './ProfessionalSummary.module.css';
+
+const ProfessionalSummary = () => {
+  const [summary, setSummary] = useState('');
+  const [error, setError] = useState('');
+
+  const validateSummary = () => {
+    if (!summary.trim()) {
+      setError('Please provide a professional summary.');
+    } else if (summary.length < 150) {
+      setError('Summary must be at least 150 characters long.');
+    } else {
+      setError('');
+    }
+  };
+
+  const handleChange = (e) => {
+    setSummary(e.target.value);
+    validateSummary();
+  };
```

Comment:
The onChange validation runs on every keystroke you could debounce it for better performance